guides/common/modules/proc_providing-feedback-on-red-hat-documentation.adoc

```diff
@@ -3,13 +3,17 @@
 [id="providing-feedback-on-red-hat-documentation_{context}"]
 = Providing feedback on Red Hat documentation
 
-We appreciate your input on our documentation.
-Please let us know how we could make it better.
+We appreciate your feedback on our documentation.
+Let us know how we can improve it.
 
-You can submit feedback by filing a ticket in Bugzilla:
+Use the Create Issue form in Red Hat Jira to provide your feedback.
```

Comment:
Is "Create Issue" a button somewhere? > make it bold? Alternative: use sentence case.

Reply:
"Create Issue" is the name of a form. When you click the long Jira link, you get to the "Create Issue" form. Since it's a UI element, it should probably be bolded.

Reply:
Done.